server/backup_runner.go

```diff
@@ -243,7 +243,7 @@ func (r *BackupRunner) downloadBinlogFilesForInstance(ctx context.Context, insta
 		return
 	}
 	mysqlDriver.SetUpForPITR(mysqlutil, binlogDir)
-	if err := mysqlDriver.FetchAllBinlogFiles(ctx, false /* downloadLatestBinlogFile */); err != nil {
+	if err := mysqlDriver.FetchAllBinlogFiles(ctx, true /* downloadLatestBinlogFile */); err != nil {
```

Comment:
How often do we download the latest file?

Reply:
10 minutes in release mode.